src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/SqlSchemaInfoTest/SqlSchemaInfoTest.cs

```diff
@@ -76,16 +76,18 @@ public static void TestCommandBuilder()
 
         // This test validates behavior of SqlInitialCatalogConverter used to present database names in PropertyGrid
         // with the SqlConnectionStringBuilder object presented in the control underneath.
-        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup))]
+        // Skip: We cannot assign an access token to the connection in this test (#31362)
+        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup), nameof(DataTestUtility.AuthenticatingWithoutAccessToken))]
         public static void TestInitialCatalogStandardValues()
         {
-            using (SqlConnection connection = new SqlConnection(DataTestUtility.TCPConnectionString))
+            Console.WriteLine($"TCPConnectionString = {DataTestUtility.TCPConnectionString}");
+
+            using (SqlConnection connection = DataTestUtility.GetSqlConnection(DataTestUtility.TCPConnectionString))
             {
                 string currentDb = connection.Database;
-                SqlConnectionStringBuilder builder = new SqlConnectionStringBuilder(connection.ConnectionString);
+                SqlConnectionStringBuilder builder = new SqlConnectionStringBuilder(DataTestUtility.TCPConnectionString);
```

Comment:
I believe the point of this test is building and reading the connection string properties from a `SqlConnection.ConnectionString`. So, instead of skipping these kind of tests you can force the using of the `DataTestUtility.TCPConnectionString` through the helper method by adding a new argument and bypassing the internal logic.
```suggestion
                SqlConnectionStringBuilder builder = new SqlConnectionStringBuilder(connection.ConnectionString);
```